test/e2e/network/endpointslice.go

```diff
@@ -528,7 +528,14 @@ var _ = common.SIGDescribe("EndpointSlice", func() {
 		gomega.Expect(epsList.Items).To(gomega.BeEmpty(), "filtered list should have 0 items")
 	})
 
-	ginkgo.It("should support a Service with multiple ports specified in multiple EndpointSlices", func(ctx context.Context) {
+	/*
+		Release: v1.34
+		Testname: EndpointSlice, single IP, multiple ports
+		Description: Given a selector-less Service with multiple manually-created
+	        EndpointSlices (and no Endpoints) where the endpoints have the same IP
```

Comment:
These spaces seem to be the cause for gofmt wanting to indent an additional level.
Locally I replaced these spaces with a tab, and gofmt was happy.

Reply:
ah. sneaky